lib/SPIRV/libSPIRV/SPIRVDecorate.h

```diff
@@ -433,6 +437,13 @@ class SPIRVGroupDecorateGeneric : public SPIRVEntryNoIdGeneric {
   }
   virtual void decorateTargets() = 0;
   _SPIRV_DCL_ENCDEC
+  void validate() const override {
+    getModule()->getErrorLog().checkError(
+        this->getModule()->getSPIRVVersion() < VersionNumber::SPIRV_1_4,
+        SPIRVEC_InvalidModule,
+        "OpGroup(Member)Decorate is deprecated starting from SPIR-V 1.4");
```

Comment:
Maybe re-write as:

"OpGroupDecorate and OpGroupMemberDecoreate are deprecated starting from SPIR-V 1.4"